autoload/neomake/makers/proselint.vim

```diff
@@ -0,0 +1,7 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#proselint#proselint() abort
+    return {
+        \ 'errorformat': '%f:%l:%c: %m'
```

Comment:
Please add the `%W` here, that was present with `neomake#makers#ft#mail#proselint`.